README.md

```diff
@@ -0,0 +1,136 @@
+### Notebook Images
+
+These images were created to be used with Open Data Hub (ODH) or Red Hat Openshift Data Science (RHODS) with the Kubeflow Notebook Controller as the launcher.
```

Comment:
I wouldn't mention RHODS in upstream repositories.

"Kubeflow Notebook Controller" should be reworded as "ODH Notebook Controller".